_posts/en/newsletters/2023-10-11-newsletter.md

```diff
@@ -0,0 +1,204 @@
+      not compile-time. It satisfies the frequent use case of wanting
+      to take input that can be either kind of identifier. For this reason,
+      this PR doesn't remove `GenTxid`. A better alternative for the
+      future might be `std::variant<Txid, Wtxid>`."
+  a5link="https://bitcoincore.reviews/28107#l-161"
+
+  q6="How is `transaction_identifier` able to subclass `uint256`, given
+      that, in C++, integers are types and not classes?"
+  a6="Because `uint256` is itself a class, rather than a built-in type.
+      (The largest built-in C++ integer type is 64 bits.)"
+  a6link="https://bitcoincore.reviews/28107#l-194"
+
+  q7="Does a `uint256` behave otherwise the same as, for example, a
+      `uint64_t?`"
+  a7="No, arithmetic operations aren't permitted on `uint256` because
+      they don't make sense for hashes (which is the main use of `uint256`).
+      The name is misleading; it's really just a blob of 256 bits.
+      A separate `arith_uint256 ` allows arithmetic (used, for example,
+      in PoW calculations)."
+  a7link="https://bitcoincore.reviews/28107#l-203"
+
+  q8="Why does `transaction_identifier` subclass `uint256` instead of
+      being a completely new type?"
+  a8="It allows us to use explicit and implicit conversions to leave
+      code that is expecting a transaction ID in the form of a `uint256`
+      unchanged until it's an appropriate time to refactor to use the new,
+      stricter `Txid` or `Wtxid` types."
+  a8link="https://bitcoincore.reviews/28107#l-219"
+%}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- FIXME:harding to update Tuesday
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo], and
+[Bitcoin Inquisition][bitcoin inquisition repo].*
+
+- [Bitcoin Core #27596][bitcoin core #27596] finishes the first phase
+  of the [assumeutxo][topic assumeutxo] project containing all the
+  remaining changes necessary to both use an assumedvalid snapshot
+  chainstate and do a full validation sync in the background. It
+  makes UTXO snapshots loadable via RPC (`loadtxoutset`) and adds
+  `assumeutxo` parameters to chainparams.
+
+    While the feature set will not be available on mainnet until
+    [activated][bitcoin core #28553], this merge marks the culmination of
+    a multi-year effort. [Proposed in 2018][assumeutxo core dev] and
+    [formalized by James O'Beirne in 2019][assumeutxo 2019 mailing list],
```

Comment:
We don't include names in the merges section of the newsletter.  The idea is that getting a PR to merge requires a team effort, so mentioning specific names without mentioning everyone involved (which is impractical for most PRs) is potentially discouraging to the unnamed contributors.

I don't know that this is really a good policy and I'm open to discussing it, but it's the policy we've used so far, so I'm going to drop O'Beirne's name from here.  (That said, what James did by continuing to work on AssumeUTXO year after year, through job transitions, jobs falling through, a pandemic, and the soul-crushing slow pace of Bitcoin Core review deserves to be celebrated.  Loudly.  Dedication like that inspires me more than I can express here.)

Reply:
>  The idea is that getting a PR to merge requires a team effort, so mentioning specific names without mentioning everyone involved (which is impractical for most PRs) is potentially discouraging to the unnamed contributors.

I wholeheartedly agree and like this policy a lot!